### PR TITLE
[17.0][MIG] account_netting: Migration to 17.0

### PR DESCRIPTION
**Pull request by MarwanBHL**
Basic Migration to 17.0

### CODE REVIEW

**Review by desdelinux — APPROVED** [review]

**Review by xmglord — APPROVED** [review]
LGTM :+1:

**Comment by desdelinux** [comment]
Hi @pedrobaeza @AaronHForgeFlow @JordiBForgeFlow this is ready :)

Could your review / merge

**Review by melanievvl — APPROVED** [review]

File: account_netting/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_account_move_make_netting,Full access on account.move.make.netting to accountant grp,model_account_move_make_netting,account.group_account_user,1,1,1,1
+access_account_move_make_netting_manager,Full access on account.move.make.netting to accountant manager grp,model_account_move_make_netting,account.group_account_manager,1,1,1,1
```
Comment (pedrobaeza) [inline account_netting/security/ir.model.access.csv:3]: Why do you need this one? The previous line applies also to this group.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: account_netting/security/ir.model.access.csv (hunk 1 above)
Comment (MarwanBHL) [reply account_netting/security/ir.model.access.csv:3]: Doesn't work if you install only this module.

Comment (pedrobaeza) [reply account_netting/security/ir.model.access.csv:3]: AFAIK, `group_account_manager` inherits from `group_account_user`, isn't it?

Comment (MarwanBHL) [reply account_netting/security/ir.model.access.csv:3]: I think that come from `account_usability` module unless I'm mistaken

Comment (MarwanBHL) [reply account_netting/security/ir.model.access.csv:3]: @pedrobaeza What do I do then? Would you prefer that I remove this line of code?

Comment (desdelinux) [reply account_netting/security/ir.model.access.csv:3]: Hi @pedrobaeza

Please give us feedback here.

We require this module in v17.0

**Review by desdelinux — APPROVED** [review]

**Review by melanievvl — APPROVED** [review]

**Review by xmglord — APPROVED** [review]

**Comment by desdelinux** [comment]
Hi @alexis-via Can you help me by merging this please?

File: account_netting/security/ir.model.access.csv (hunk 1 above)
Comment (pedrobaeza) [reply account_netting/security/ir.model.access.csv:3]: OK, the existing permission was "Technical / Show Full Accounting Features", and this one is adding "Invoicing / Billing Administrator" to do the netting. It's a change of scope, but I think is correct.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**